x-pack/plugins/uptime/server/lib/synthetics_service/formatters/format_configs.test.ts

```diff
@@ -13,127 +14,158 @@ import {
+    let testBrowserConfig: Partial<MonitorFields>;
+    let formattedBrowserConfig: Record<string, FormattedValue>;
+
+    beforeEach(() => {
+      testBrowserConfig = {
+        type: DataStream.BROWSER,
+        enabled: true,
+        schedule: { number: '3', unit: ScheduleUnit.MINUTES },
+        'service.name': '',
+        tags: [],
+        timeout: '16',
+        name: 'Test',
+        locations: [],
+        __ui: {
+          script_source: { is_generated_script: false, file_name: '' },
+          is_zip_url_tls_enabled: false,
+          is_tls_enabled: false,
+        },
+        'source.zip_url.url': '',
+        'source.zip_url.username': '',
+        'source.zip_url.password': '',
+        'source.zip_url.folder': '',
+        'source.zip_url.proxy_url': '',
+        'source.inline.script':
+          "step('Go to https://www.google.com/', async () => {\n  await page.goto('https://www.google.com/');\n});",
+        params: '',
+        screenshots: 'on',
+        synthetics_args: [],
+        'filter_journeys.match': '',
+        'filter_journeys.tags': ['dev'],
+        ignore_https_errors: false,
+        'throttling.is_enabled': true,
+        'throttling.download_speed': '5',
+        'throttling.upload_speed': '3',
+        'throttling.latency': '20',
+        'throttling.config': '5d/3u/20l',
+      };
+
+      formattedBrowserConfig = {
+        enabled: true,
+        'filter_journeys.tags': ['dev'],
+        ignore_https_errors: false,
+        locations: [],
+        name: 'Test',
+        schedule: '@every 3m',
+        screenshots: 'on',
+        'source.inline.script':
+          "step('Go to https://www.google.com/', async () => {\n  await page.goto('https://www.google.com/');\n});",
+        throttling: '5d/3u/20l',
+        timeout: '16s',
+        type: 'browser',
+      };
     });
 
-    testBrowserConfig['throttling.is_enabled'] = false;
-    testBrowserConfig['filter_journeys.tags'] = [];
+    it('sets browser keys properly', () => {
+      const yamlConfig = formatMonitorConfig(
+        Object.keys(testBrowserConfig) as ConfigKey[],
+        testBrowserConfig
+      );
 
-    expect(
-      formatMonitorConfig(Object.keys(testBrowserConfig) as ConfigKey[], testBrowserConfig)
-    ).toEqual({
-      enabled: true,
-      locations: [],
-      name: 'Test',
-      schedule: '@every 3m',
-      screenshots: 'on',
-      'source.inline.script':
-        "step('Go to https://www.google.com/', async () => {\n  await page.goto('https://www.google.com/');\n});",
-      throttling: false,
-      timeout: '16s',
-      type: 'browser',
+      expect(yamlConfig).toEqual(formattedBrowserConfig);
+    });
+
+    it('excludes UI fields', () => {
+      testBrowserConfig['throttling.is_enabled'] = false;
+
+      const formattedConfig = formatMonitorConfig(
+        Object.keys(testBrowserConfig) as ConfigKey[],
+        testBrowserConfig
+      );
+
+      expect(formattedConfig).toEqual(formattedConfig);
```

Comment:
Is this expect supposed to look at a different value?

Reply:
Yes, it is. Thanks for the catch!